#source/scss/style.scss

```diff
@@ -0,0 +1,98 @@
+@import "nullstyle.scss";
+
+
+ body{
+ 	margin: 0 auto;
+ 	font-family: 'Montserrat', sans-serif;
+ }
+ @import "../fonts/font.css";
+
+ // ======================================= header
+.header {
+	padding: 24px 0 0 0; 

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/chendler_tesla/pullRequest?prid=5633737)